Dockerfiles/manifests/rbac/clusterrole.yaml

```diff
@@ -16,6 +16,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups: ["quota.openshift.io"]
```

Comment:
Should this be commented out here? What happens if someone tries to create this manifest that isn't using openshift?

Reply:
Unknown `apigroups` or `resources` are silently ignored, so we're not triggering an error on vanilla k8s.

Indeed, we could split these in a separate `clusterrole`+`clusterrolebinding` pair on the same `serviceaccount`, I'm concerned about lengthening the setup instructions though.